framework/core/js/src/common/states/PaginatedListState.ts

```diff
@@ -121,6 +121,10 @@ export default abstract class PaginatedListState<T extends Model, P extends Pagi
       include,
     };
 
+    if (!params.include) {
+      delete params.include;
```

Comment:
Why is this necessary? If a falsy value is provided, should we just ignore it on the server?

Reply:
good catch, `include=` should not be ignored, but this is meant to prevent the case where no import is provided in the fontend code, since that would still send an empty `include` to the server and the server interprets it as no includes. when in reality the frontend just didn't have any developer-specified incudes which isn't the same as empty includes

changed to
```js
    if (typeof params.include === 'undefined') {
      delete params.include;
    }
```